docs/docs/resources/pipeline-components/sections/repo_config-kubernetes-app.yaml

```diff
@@ -0,0 +1,10 @@
+  # Helm repository configuration (optional)
+  # If not set the helm repo add will not be called. Useful when using local Helm charts
+  repoConfig:
+    repositoryName: bakdata-streams-bootstrap # required
```

Comment:
Didn't we agree on snake_case for KPOps models?

Reply:
that's moved to v2